plugins/pulp_docker/plugins/migrations/0004_tag_schema_field_change.py

```diff
@@ -0,0 +1,16 @@
+from pulp.server.db.connection import get_collection
+
+
+def migrate(*args, **kwargs):
+    """
+    Changes chema_version to string in the tag collection.
+    """
+
+    schema_version_key = 'schema_version'
+    collection = get_collection('units_docker_tag')
+    # update collection
+    for tag in collection.find():
```

Comment:
It's tempting to run two multi-update commands and call it a day, so there's no iteration in python, nor transfer of objects from the DB into python. Something like:
```
collection.update({schema_version_key: 1}, {'$set': {schema_version_key: '1'}, {'multi':true}})
collection.update({schema_version_key: 2}, {'$set': {schema_version_key: '2'}, {'multi':true}})
```